exercises/concept/karls-languages/src/test/java/LanguageListTest.java

```diff
@@ -70,18 +62,48 @@ public void countThree() {
     }
 
     @Test
+    @Tag("task:5")
+    @DisplayName("The count method returns 0 in case no language was added to the list")
     public void countEmpty() {
         assertThat(languageList.count()).isEqualTo(0);
     }
 
     @Test
+    @Tag("task:6")
+    @DisplayName("The containsLanguage method returns the correct booleans after adding a language to the list")
+    public void addOneLanguage() {
+        languageList.addLanguage("Java");
+
+        assertThat(languageList.containsLanguage("Java")).isTrue();
+        assertThat(languageList.containsLanguage("Python")).isFalse();
+    }
+
+    @Test
+    @Tag("task:6")
+    @DisplayName("The containsLanguage method returns the correct booleans after adding several languages to the list")
+    public void addMultipleLanguages() {
+        languageList.addLanguage("Java");
+        languageList.addLanguage("Ruby");
+        languageList.addLanguage("C++");
+
+        assertThat(languageList.containsLanguage("Java")).isTrue();
+        assertThat(languageList.containsLanguage("Ruby")).isTrue();
+        assertThat(languageList.containsLanguage("C++")).isTrue();
+        assertThat(languageList.containsLanguage("Python")).isFalse();
+    }
```

Comment:
I would suggest refactoring these tests to make them a bit simpler, something like this:

```java
@Test
@Tag("task:6")
@DisplayName("The containsLanguage returns true when the language is in the list")
public void containsLanguage() {
     languageList.addLanguage("Java");

    assertThat(languageList.containsLanguage("Java")).isTrue();
}

@Test
@Tag("task:6")
@DisplayName("The containsLanguage returns false when the language is not in the list")
public void doesNotContainLanguage() {
    languageList.addLanguage("Kotlin");

    assertThat(languageList.containsLanguage("Java")).isFalse();
}
```